src/h3lib/include/polygon.h

```diff
@@ -48,7 +48,8 @@ typedef enum {
     CONTAINMENT_CENTER = 0,       ///< Cell center is contained in the shape
     CONTAINMENT_FULL = 1,         ///< Cell is fully contained in the shape
     CONTAINMENT_OVERLAPPING = 2,  ///< Cell overlaps the shape at any point
-    CONTAINMENT_INVALID = 3  ///< This mode is invalid and should not be used
+    CONTAINMENT_OVERLAPPING_BBOX = 3,  ///< Cell bounding box overlaps shape
```

Comment:
Is this purely an internal detail, or should be exposed to the user?

Reply:
I think users could potentially benefit from this - it's a faster (2-3x) version of `CONTAINMENT_OVERLAPPING` that may contain false positives, but no false negatives. Not appropriate for most use cases, but some users might find it useful.